packages/social-metadata-previews/src/facebook/FacebookDescription.js

```diff
@@ -1,26 +1,6 @@
 /* External dependencies */
 import styled from "styled-components";
 
-/**
- * Determines the number of lines depending on the imageMode.
- *
- * @param {String} mode The imageMode.
- *
- * @returns {Number} Number of lines the description shows.
- */
-const determineClamp = ( mode ) => {
-	switch ( mode ) {
-		case "landscape":
-			return 1;
-		case "square":
-			return 3;
-		case "portrait":
-			return 5;
-		default:
-			return 1;
-	}
-};
-
 // Used to make sure the element also has a height when empty by setting min-height equal to line-height.
 const height = "16px";
```

Comment:
Maybe take the same approach as you did for `title` here? So rename to something more specific and make it a `number` type.